### DISCUSSION TITLE
Videos not showing up when deployed to gh pages.

### DISCUSSION THREAD

**Discussion by Harrman17** [community/community#133137] [Programming Help]
### Select Topic Area

Question

### Body

I have a bunch of videos that go with my React project, everything else is there on gh pages including the folder that contains the videos but they do not show up on the website.

How do I go about fixing this? Here is the repository: https://github.com/Harrman17/MotivationWatch

**Comment by knarrff** [comment]
Hi @Harrman17 

Your videos are there, e.g.,

https://harrman17.github.io/MotivationWatch/MotivationWatchVids/10years.mp4

However, within the page source, they are linked like this:

https://harrman17.github.io/MotivationWatchVids/embracethestruggle.mp4

(note the missing 'MotivationWatch/').

Does this help?

**Reply by knarrff** [reply]
You might be able to fix this by correcting the "homepage" link in package.json by appending the "/MotivationWatch": https://github.com/Harrman17/MotivationWatch/blob/5fc67fd6ebbf4f2532c554a08d7c32045bfe3954/package.json#L2

**Reply by Harrman17** [reply]
Nope this does not work

**Comment by LiteBrite82** [comment]
app.jsx line 7 `import VideoLinksFile from '/src/VideoLinks.json'`, the path you are using is incorrect. You do not need to include /src

**Reply by Harrman17** [reply]
still does not work, im pretty sure its the same thing

**Reply by LiteBrite82** [reply]
I'm gonna move this over to `Programming Help` to see if we can get some more exposure.
